helper-cli/src/main/kotlin/commands/MergeAnalyzerResultsCommand.kt

```diff
@@ -0,0 +1,291 @@
+        .convert { it.absoluteFile.normalize() }
+        .required()
+
+    override fun run() {
+        val inputOrtResults: MutableList<AnalyzerRun> = LinkedList()
+        val inputRepositories: MutableList<Repository> = LinkedList()
+
+        inputAnalyzerResultFiles.stream()
+            .map { it.readValue<OrtResult>() }
+            .forEach {
+                it.analyzer.let { value -> inputOrtResults.add(value!!) }
+                inputRepositories.add(it.repository)
+            }
+
+        val analyzerRun = AnalyzerRun(
+            startTime = now(),
+            endTime = now(),
+            environment = Environment(),
+            config = aggregateAnalyzerConfiguration(inputOrtResults),
+            result = aggregrateAnalyzerRun(inputOrtResults)
+        )
+
+        val repository = Repository(
+            vcs = analyzerRun.result.projects.first().vcs,
+            vcsProcessed = analyzerRun.result.projects.first().vcsProcessed,
+            config = aggregateRepositoryConfigurations(inputRepositories)
+        )
+
+        val outputFile = OrtResult(
+            repository = repository,
+            analyzer = analyzerRun
+        )
+
+        outputAnalyzerResultFile.writeValue(outputFile)
+    }
+
+    private fun aggregateRepositoryConfigurations(repositories: List<Repository>): RepositoryConfiguration {
+        fun mergeExcludes(leftExlcudes: Excludes, rightExcludes: Excludes): Excludes = Excludes(
+            paths = (leftExlcudes.paths + rightExcludes.paths).distinct(),
+            scopes = (leftExlcudes.scopes + rightExcludes.scopes).distinct()
+        )
+
+        fun mergeResolutions(leftResolutions: Resolutions, rightResolutions: Resolutions): Resolutions =
+            Resolutions(
+                issues = (leftResolutions.issues + rightResolutions.issues).distinct(),
+                ruleViolations = (leftResolutions.ruleViolations + rightResolutions.ruleViolations).distinct(),
+                vulnerabilities = (leftResolutions.vulnerabilities + rightResolutions.vulnerabilities).distinct()
+            )
+
+        fun mergeCurations(leftCurations: Curations, rightCurations: Curations): Curations =
+            Curations(
+                licenseFindings = (leftCurations.licenseFindings + rightCurations.licenseFindings).distinct()
+            )
+
+        fun mergeLicenseChoices(leftChoices: LicenseChoices, rightChoices: LicenseChoices): LicenseChoices =
+            LicenseChoices(
+                repositoryLicenseChoices = (
+                        leftChoices.repositoryLicenseChoices
+                                + rightChoices.repositoryLicenseChoices
+                        ).distinct(),
+                packageLicenseChoices = (
+                        leftChoices.packageLicenseChoices
+                                + rightChoices.packageLicenseChoices
+                        ).distinct()
```

Comment:
Merging license choices can lead to unexpected results, if they are contradictory, or if license choices from one analyzer result apply to findings from another analyzer result.
There are more similar consistency issues in the code below where data is lost because the model was not designed for this use case.

Reply:
This might be true but we chose to leave that problem to the analyst to sort it out.

Anyway, it is bad sign if project team choose contradicting license statements